src/service/cid-impl.js

```diff
@@ -50,6 +50,11 @@ const BASE_CID_MAX_AGE_MILLIS = 365 * ONE_DAY_MILLIS;
 
 const SCOPE_NAME_VALIDATOR = /^[a-zA-Z0-9-_.]+$/;
 
+const CID_OPTOUT_STORAGE_KEY = 'amp-cid-optout';
+
+// TODO(aghassemi): replace with real viewer message when implemented.
+const CID_OPTOUT_VIEWER_MESSAGE = 'cidOptout';
```

Comment:
@lannka Does viewer already have support for sending a message to inform user has opted out? For now I am just sending a message with this key